src/coreclr/gc/gcpriv.h

```diff
@@ -4132,10 +4126,6 @@ class gc_heap
     size_t committed_by_oh_per_heap[total_oh_count];
 #endif // _DEBUG && MULTIPLE_HEAPS
 
-    // This is what GC uses for its own bookkeeping.
-    PER_HEAP_ISOLATED
-    size_t current_total_committed_bookkeeping;
```

Comment:
This field was there before I introduced `heap_hard_limit_for_bookkeeping`.

Reply:
Oops, ok, I'll put it back...